android/src/main/java/jp/manse/BrightcovePlayerView.java

```diff
@@ -70,16 +84,55 @@ public BrightcovePlayerView(ThemedReactContext context, AttributeSet attrs) {
 
     }
 
+    private void setupProgress(ThemedReactContext context){
+        this.progressBar = new ProgressBar(context);
+        this.progressBar.setIndeterminate(true);
+
+        int RADIUS_DP = 16;
+        int WIDTH_DP = 4;
+
+        final DisplayMetrics metrics = getResources().getDisplayMetrics();
+        final float screenDensity = metrics.density;
+        CircularProgressDrawable drawable = new CircularProgressDrawable(context);
+        drawable.setColorSchemeColors(Color.WHITE);
+        drawable.setCenterRadius(RADIUS_DP * screenDensity);
+        drawable.setStrokeWidth(WIDTH_DP * screenDensity);
+        this.progressBar.setIndeterminateDrawable(drawable);
+        this.progressBar.setVisibility(VISIBLE);
+    }
+
+    private int getBitrate(){
+        SharedPreferences pref = getContext().getSharedPreferences(TAG, 0);
+        if (pref.contains(BITRATE)) {
+            return pref.getInt(BITRATE,Integer.MAX_VALUE);
+        } else {
+            return Integer.MAX_VALUE;
```

Comment:
I just updated the AC.  Thinking about it, it makes sense to start with 3,500,000 i.e. an intermediate value if we don't know anything about the available bandwidth.  BC will adapt up to max if that works

Reply:
Yep fixed